src/rgw/rgw_rest_s3.cc

```diff
@@ -6315,6 +6521,7 @@ int RGWSelectObj_ObjStore_S3::handle_aws_cli_parameters(std::string& sql_query)
 
   extract_by_tag("QuoteEscapeCharacter", m_escape_char);
   extract_by_tag("CompressionType", m_compression_type);
+  extract_by_tag("Enabled", m_enabled);
```

Comment:
please verify (review the client request) that the tag "Enabled" indeed exists in request.

Reply:
Yes, it works fine while testing with aws cli( --request-progress '{"Enabled": true}')